resources/assets/lib/user-card.tsx

```diff
@@ -157,17 +159,25 @@ export class UserCard extends React.PureComponent<PropsInterface, StateInterface
   }
 
   renderMenuButton() {
-    if (!this.canMessage) { return null; }
+    if (this.isSelf) { return null; }
 
     const items = (dismiss: () => void) => (
       <>
-        <a
-          className='simple-menu__item js-login-required--click'
-          href={laroute.route('messages.users.show', { user: this.user.id })}
-          onClick={dismiss}
-        >
-          {osu.trans('users.card.send_message')}
-        </a>
+        {
+          this.canMessage ? (
```

Comment:
just noticed this checks for user blocks but no popup is shown for blocked users anyway

Reply:
cards already created for a user still show even after they're blocked